### ISSUE TITLE
I need a 64 bit version of MAME4iOS that still had the wiimote option in BTStack

### ISSUE THREAD

**Issue by Thekidiacs** [yoshisuga/MAME4iOS#413]
I am trying to add wiimote support for later iOSes. It is critical that I can get this version. Compiling the original seleuco version was an absolute nightmare and I couldn't do it. Please if someone could provide me the deb for it I would be so grateful. Thanks.

**Comment by Thekidiacs** [comment]
@ToddLa @yoshisuga

**Comment by ToddLa** [comment]
The WiMote required a jailbroken device and a hacked BTStack. 

Your best plan is to build the mame4all project.

**Comment by Thekidiacs** [comment]
Yes, I have multiple jailbroken devices on multiple iOSes for testing. Does the old versions of this repository from yoshisuga have the old btstack code with wiimote support from earlier commits like from 2016-2017?

**Comment by Alexck002** [comment]
> provide

Which .deb file are you requiring? Btw that old version should be the one from 2013
